src/Illuminate/Queue/Worker.php

```diff
@@ -148,7 +148,7 @@ protected function daemonShouldRun()
      */
     protected function waitForChildProcess($processId, $timeout)
     {
-        declare(ticks=1) {
+        declare (ticks = 1) {
```

Comment:
@GrahamCampbell this change was made when I ran `php-cs-fixer fix` on my machine, but it breaks on StyleCI.